components/addForm.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+
+export default function AddCharacterForm() {
```

Comment:
Please name the function identically to the name of the file: 'AddForm', or rename the file accordingly. And the file names of the components (not of the pages!) should start with an uppercase letter.